### PR TITLE
added another realization of buzzard v2.0

### PR DESCRIPTION
**Pull request by j-dr**
I've added a config for another realization of buzzard v2.0

### CODE REVIEW

File: GCRCatalogs/catalog_configs/buzzard_v2.0.0_4.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+subclass_name: buzzard.BuzzardGalaxyCatalog
+
+catalog_root_dir: /global/cfs/cdirs/lsst/shared/external/buzzard/Buzzard-4/v2.0.0/
```
Comment (yymao) [inline GCRCatalogs/catalog_configs/buzzard_v2.0.0_4.yaml:3]:
```suggestion
catalog_root_dir: ^/external/buzzard/Buzzard-4/v2.0.0/
```

**Comment by j-dr** [comment]
That sounds good to me. I think anyone who needs this catalog urgently can just use my fork for now.

**Comment by yymao** [comment]
@JoanneBogart let's merge this into your branch first so we can test it. Merge at your leasure.

File: GCRCatalogs/catalog_configs/buzzard_v2.0.0_4.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    H0: 70.0
+    Om0: 0.286
+    Ob0: 0.047
+    sigma8: 0.82
+    n_s: 0.96
+
+halo_mass_def: vir
+lightcone: true
+sky_area : 10313.24
+version: '2.0.0_3'
```
[the remark this answers is not in the document]
Comment (yymao) [reply GCRCatalogs/catalog_configs/buzzard_v2.0.0_4.yaml:18]: Good call @cosmicshear 
```suggestion
version: '2.0.0_4'
```

[the remark this answers is not in the document]
Comment (yymao) [reply GCRCatalogs/catalog_configs/buzzard_v2.0.0_4.yaml:18]: Oops. Just realize this has been merged. I'll need to open a new PR.

**Pull request merged**